src/lib/Ouroboros/Byron/Proxy/Main.hs

```diff
@@ -510,7 +508,8 @@ bbsStreamBlocks rr idx db _ hh f yieldFirst k = do
             _ -> error "bbsStreamBlocks: ChainDB reader unexpected instruction"
   where
 
-  acquireReader = ChainDB.deserialiseReader <$> ChainDB.newBlockReader db rr
+  --acquireReader = ChainDB.deserialiseReader <$> ChainDB.newBlockReader db rr
```

Comment:
Why leave in the old code? (same comment applies to a few other places)

Reply:
Just forgot to delete.